DDDEastAnglia.Tests/Sponsors/sponsor_list/Sponsors_who_havent_paid_yet.cs

```diff
@@ -0,0 +1,31 @@
+using System.Linq;
+using NUnit.Framework;
+
+namespace DDDEastAnglia.Tests.Sponsors.sponsor_list
+{
+    [TestFixture]
+    public sealed class Sponsors_who_havent_paid_yet : context
+    {
+        public Sponsors_who_havent_paid_yet()
```

Comment:
Yes, that's a fair point: Xunit gives you an instance per test (one of the reasons it's better than NUnit 😆)

Second opinion, @adrianbanks?